src/pages/dashboard/_UsCumulativeDeathsContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```

Comment:
In general, let's try and rename these to just kebab-case-files. Also, the underscore should be removed.